backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/image/ImagesHandler.java

```diff
@@ -515,18 +515,30 @@ private DiskImage isImageExist(Guid storagePoolId, DiskImage image) {
     }
 
     public static boolean checkImageConfiguration(StorageDomainStatic storageDomain,
-            DiskImageBase diskInfo, List<String> messages) {
-        if (!checkImageConfiguration(storageDomain, diskInfo.getVolumeType(), diskInfo.getVolumeFormat(), diskInfo.getBackup())) {
+                                                  VolumeType volumeType,
+                                                  VolumeFormat volumeFormat,
+                                                  DiskBackup diskBackup,
+                                                  List<String> messages) {
```

Comment:
did you apply the formatter?

Reply:
I did and that's what I got, I guess it's misconfigured or something, I'll push it back manually